coffee_cmd/src/main.rs

```diff
@@ -38,7 +38,13 @@ async fn main() -> Result<(), CoffeeError> {
             }
             result
         }
-        CoffeeCommand::Remove => todo!(),
+        CoffeeCommand::Remove { plugin } => match coffee.remove(&plugin).await {
+            Ok(_) => {
+                term::success!("successfully uninstalled plugin {plugin}");
+                Ok(())
+            }
+            Err(err) => Err(err),
+        },
```

Comment:
can you follow the coffee_term style that I added in a previous PR?

Reply:
Done